packages/manager/apps/overthebox/package.json

```diff
@@ -2,6 +2,11 @@
   "name": "@ovh-ux/manager-overthebox-app",
   "version": "5.0.0",
   "private": true,
+  "repository": {
```

Comment:
What does this have to do with this PR ?

Reply:
The `repository` entry is needed to be able to display all components
and associated them to their corresponding location.